hms-app/src/pages/MyIdeas.tsx

```diff
@@ -155,6 +158,7 @@ export default function MyIdeas() {
                   columnSize={6}
                   type={IdeaCardType.Owner}
                   isLoading={false}
+                  reloadIdeaList={loadSelectedHackathon}
```

Comment:
could you name this "OnSuccess"?

Reply:
that would require me to rename the props in `IdeaCardList`
Does anyone currently working with that?

Reply:
@rz-p you change the props to `onSuccess`, but haven't change it in the `IdeaCardList`(?)
Then you also need to change it in a few components that uses `IdeaCardList`